src/python/WMComponent/JobUpdater/JobUpdaterPoller.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, config):
         self.config = config
 
         self.bossAir = BossAirAPI(config = self.config)
-        self.reqmgr = RequestManager({'endpoint' : self.config.JobUpdater.reqMgrUrl})
+        self.reqmgr = ReqMgr({'endpoint' : self.config.JobUpdater.reqMgrUrl})
```

Comment:
This one only has read call, so we can only reqmgr2 and remove the reqmgr call completely. But for the clarity, I should change the config name. I will do that.